unittests/test_kernels.py

```diff
@@ -279,28 +279,28 @@ def runTest(self):
 
     #### ckcov
     values = ckcov(PATH_ + '17257_17262ra.bc', -82000, False, 'INTERVAL', 1., 'SCLK')
-    self.assertEqual(values[0,0], 304593554335.0)
-    self.assertEqual(values[0,1], 304610188193.0)
-    self.assertEqual(values[1,0], 304610195359.0)
-    self.assertEqual(values[1,1], 304622337377.0)
-    self.assertEqual(values[2,0], 304622354271.0)
-    self.assertEqual(values[2,1], 304625376609.0)
+    self.assertEqual(values[0], 304593554335.0)
```

Comment:
I don't understand. Why is this an improvement? I thought have the min/max values as pairs was a lot more obvious to use. Or should this be using `as_invervals`?